package.json

```diff
@@ -24,7 +24,8 @@
     "pretest": "npm run lint:frontend",
     "test": "jest app/javascript/ bin/ --coverage",
     "test:watch": "jest app/javascript/ bin/ --watch",
-    "postcss": "postcss public/assets/*.css -d public/assets 2> postcss_error.log"
+    "postcss": "postcss public/assets/*.css -d public/assets 2> postcss_error.log",
+    "e2e": "cypress open"
```

Comment:
`yarn e2e` will run Cypress for local development. CI will come in a separate PR including integration with Knapsack.